charts/stardog/templates/statefulset.yaml

```diff
@@ -144,24 +156,33 @@ spec:
       - name: stardog-license
         secret:
           secretName: stardog-license
-      - name: {{ include "stardog.fullname" . }}-config
+      - name: {{ include "stardog.fullname" . }}-properties-vol
         configMap:
-          name: {{ include "stardog.fullname" . }}
+          name: {{ include "stardog.fullname" . }}-properties
+          items:
+          - key: stardog.properties
+            path: stardog.properties
+      - name: {{ include "stardog.fullname" . }}-log4j-vol
+        configMap:
+          name: {{ include "stardog.fullname" . }}-log4j
+          items: 
+          - key: log4j2.xml
+            path: log4j2.xml
       - name: {{ include "stardog.fullname" . }}-password
         secret:
           secretName: {{ include "stardog.fullname" . }}-password
           items:
             - key: password
               path: adminpw
-              mode: 400
+              mode: 444
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 8 }}
   volumeClaimTemplates:
   - metadata:
       name: data
     spec:
-      accessModes: [ "ReadWriteOnce" ]
-      storageClassName: {{ .Values.persistence.storageClass }}
+      accessModes: [ "ReadWriteOnce" ]{{ if .Values.persistence.storageClass }}
```

Comment:
this might be a breaking change for any users you've got. they'll need to specify "standard", if they've had your default.

Reply:
so specifically this will help for anyone who just wants to stand it up on k8s on their local docker-for-desktop for testing. on both windows and mac it seems 'standard' is not installed, by default. so without it specified, it will just fall-back to whatever IS standard. 
actually, I think (but am not certain) that the OS on GCP w/GCE-disk didn't install 'standard' either.